charts/kube-prometheus-stack/templates/exporters/kubelet/servicemonitor.yaml

```diff
@@ -112,6 +122,8 @@ spec:
     path: {{ .Values.kubelet.serviceMonitor.resourcePath }}
```

Comment:
If cadvisor and /metrics/resource shares the same logic, the `if .Values.kubelet.serviceMonitor.trackTimestampsStaleness` should be added to the resource metrics endpoint, not to the probes endpoint.